drivers/volume/portworx/schedops/k8s-schedops.go

```diff
@@ -67,6 +67,11 @@ const (
 	talismanServiceAccount      = "talisman-account"
 	talismanImage               = "portworx/talisman:latest"
 	rancherControlPlaneLabelKey = "node-role.kubernetes.io/controlplane"
+
+	// pureVolumeOUID is used to identify if a mapper device is of Pure origin
+	// (if it leads with /dev/mapper/3624a9370, it's a Pure volume, if it's some other prefix it's not)
+	pureVolumeOUID = "3624a9370"
```

Comment:
Is this always true or specific to an array? What is special about this number?

Reply:
This is always true for Pure devices: it's our OUID, our company-specific identifier. It's also what we use in Portworx for FA cloud drives.